README.md

```diff
@@ -68,3 +68,23 @@ Shutdown Instructions (Optional)
 3. Access your zookeeper instance: `/PATH/TO/zookeeper/bin/zkCli.sh`
 4. Remove hdfs-mesos framework state from zookeeper: `rmr /hdfs-mesos`
 5. (Optional) Clear your data directories as specified in your `mesos-site.xml`. This is necessary to relaunch HDFS in the same directory.
+
+Docker Usage (Optional)
+--------------------------
+
+### Use [Official Mesosphere Image](https://registry.hub.docker.com/u/mesosphere/hdfs-mesos/)
```

Comment:
I will add information to dockerhub with a description and a link to the github repo. I will also create a JIRA to create an automated build to create the docker for each release.